RIOT-OS/RIOT
# boards/common/blxxxpill: Update pinout

### Contribution description

Also document the OSC32 connection of PC14/PC15 in the pinout.

### Testing procedure

Check the generated doc; or check the SVG updated by this PR (Github does have a nice SVG viewer inline that can show old and new versions side by side).

### Issues/PRs references

None
